scripts/run_tasks.py

```
@@ -0,0 +1,196 @@
+
+    file_path = args.file
+    retries = args.retries
+
+    tasks = parse_tasks(file_path)
+    uncompleted = find_uncompleted(tasks)
+
+    if not uncompleted:
+        print("No uncompleted tasks found. Nothing to do.")
+        return 0
+
+    for idx, task in enumerate(uncompleted):
```

Ruff (0.14.3)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)